addons/web/static/tests/views/fields/ace_editor_field_tests.js

```diff
@@ -96,4 +96,30 @@ QUnit.module("Fields", (hooks) => {
 
         assert.ok(target.querySelector(".o_field_ace").textContent.includes("blip"));
     });
+
+    QUnit.test("unset value and change pager", async (assert) => {
```

Comment:
maybe find a better test name, i.e. "leave an untouched record with an unset ace field should not write"